src/main/java/org/prebid/cache/config/StorageConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.prebid.cache.config;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Configuration
+@Validated
```

Comment:
this one is not utilized, or some validation annotations are missing

Reply:
Nice catch, changed that!